Makefile

```diff
@@ -71,7 +71,7 @@ IS_WSL := $(shell uname -r | grep -i microsoft)
 ifeq ($(IS_WSL),)
 get_ip = $(shell ifconfig | awk '/^[a-z0-9]+: /{iface=substr($$1, 1, length($$1)-1)} /status: active/{print iface}' | head -1 | xargs -I {} ifconfig {} | awk '/inet /{print $$2; exit}')
 else
-get_ip = $(shell awk '/nameserver / {print $$2; exit}' /etc/resolv.conf)
+get_ip = $(shell ifconfig en0 | grep inet | awk '$$1=="inet" {print $$2}')
```

Comment:
Can we verify that this change will work for developers with Linux and someone else with Mac?

Reply:
For mac it is verified with Tomáš Chlebek too, and I will try to find someone with linux to verify as well.

Reply:
Never mind, I will revert this change as the former is actually working on macos and linux as well.